<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'ps_checkout.event.dispatcher' shared service.

return $this->services['ps_checkout.event.dispatcher'] = new \PrestaShop\Module\PrestashopCheckout\Event\SymfonyEventDispatcherAdapter(($this->services['ps_checkout.event.dispatcher.factory'] ?? $this->load('getPsCheckout_Event_Dispatcher_FactoryService.php'))->create([0 => ($this->services['ps_checkout.event.subscriber.checkout'] ?? $this->load('getPsCheckout_Event_Subscriber_CheckoutService.php')), 1 => ($this->services['ps_checkout.event.subscriber.order'] ?? $this->load('getPsCheckout_Event_Subscriber_OrderService.php')), 2 => ($this->services['ps_checkout.event.subscriber.paypal.order'] ?? $this->load('getPsCheckout_Event_Subscriber_Paypal_OrderService.php')), 3 => ($this->services['ps_checkout.event.subscriber.paypal.capture'] ?? $this->load('getPsCheckout_Event_Subscriber_Paypal_CaptureService.php'))]));
